src/Security/Authentication/test/CertificateTests.cs

```diff
@@ -928,39 +931,110 @@ public async Task VerifyValidationResultNeverCachedAfter30Min(bool cache)
 
     private static class Certificates
     {
-        public static X509Certificate2 SelfSignedPrimaryRoot { get; private set; } =
-            new X509Certificate2(GetFullyQualifiedFilePath("validSelfSignedPrimaryRootCertificate.cer"));
```

Comment:
Can validSelfSignedPrimaryRootCertificate.cer be removed yet?

Reply:
You mean the actual files?  I'm a bit wary about those since those are in a shared folder as opposed to when they were just in the security tests

Reply:
Looks like its only Microsoft.AspNetCore.HttpOverrides.Tests which also uses some of these

```Middleware\HttpOverrides\test\Microsoft.AspNetCore.HttpOverrides.Tests.csproj:10:    <Content Include="$(SharedSourceRoot)test\Certificates\*.cer" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" />
```

Reply:
Actually I guess I can update those tests too and then just nuke the entire directory

Reply:
Updated and moved the Certificates helper to that shared directory instead and have HttpOverrides reuse the same class which let me nuke all the cer files hopefully